core/services/cable_guy/api/manager.py

```diff
@@ -48,6 +50,10 @@ class EthernetManager:
     def __init__(self, default_config: EthernetInterface) -> None:
         self.settings = settings.Settings()
 
+        self._config_path = pathlib.Path(__file__).parent.absolute().joinpath("settings", "dnsmasq.conf")
+        self._server = Dnsmasq(self._config_path)
+        self._dhcp_server_gateway = "192.168.2.2"
```

Comment:
I think that we should allow any IP address but have this one as default for bluerov and frontend.

Reply:
I think that would be cool, but this will add some good work and test.

Can we open an enhancement issue to track that and postpone it? I would like to merge the DHCP server as it is, as it adds value already, and focus on the firmware management and frontend for now.

Reply:
Opened #336 to track custom configurations.